### PR TITLE
SCTTR-173: add TikTok as source-medium and add order and consistency to source options

### PR DESCRIPTION
**Pull request by nicduke38degrees**
https://38deg.atlassian.net/browse/SCTTR-173

### Testing
I deployed to staging to test
Screenshot 2023-08-22 at 15 27 58

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ Metrics/MethodLength:
 
 Metrics/BlockLength:
   Max: 30
+  CountAsOne: ['hash']
```
Comment (nicduke38degrees) [inline .rubocop.yml:26]: This tells rubocop to stop copping on long hash literals. Like `config.source_mediums`

File: config/initializers/darjeelink.rb [hunk 2]
```diff
@@ -5,29 +5,31 @@
   config.domain = ENV['DOMAIN']
 
   config.source_mediums = {
+    chatbot: 'Chatbot',
```
Comment (nicduke38degrees) [inline config/initializers/darjeelink.rb:9]: Ordered all options alphabetically 
Apologies for refactor at same time as feature change, but the Github diff is correctly differentiating what's just moved up/down and what's new (almost)

File: .rubocop.yml (hunk 1 above)
Comment (Tim-Langford) [reply .rubocop.yml:26]: Cool. Did not know that, but believe you.

File: config/initializers/darjeelink.rb (hunk 2 above)
Comment (Tim-Langford) [reply config/initializers/darjeelink.rb:9]: Cant think of much to add here. If we do not sort it automatically, then maybe worth adding a comment to explain if people add to it we want to keep it lexicographically sorted?

Obvs dont really matter... just a suggestion.

**Review by Tim-Langford — APPROVED** [review]
LGTM

File: config/initializers/darjeelink.rb (hunk 2 above)
Comment (nicduke38degrees) [reply config/initializers/darjeelink.rb:9]: Good suggestion 🤔 
idk if sorting just before rendering is smarter or unnecessary 🤷 
I quite like the convention that all collections in code (as in array/hash/object literals) are sorted alphabetically by default, unless there's goo reason not too. Make it easier to read imo.

I'll a comment in an attempt to preserve future ordering - hopefully subsequent PRs to this area can keep this in line too

**Pull request merged**